Source/Data/BPMFBase.txt

```diff
@@ -331,6 +331,7 @@
 鸔 ㄅㄛˊ bo2 1i6 big5
 糪 ㄅㄛˊ bo2 1i6 big5
 蔔 ㄅㄛ˙ bo5 1i7 big5
+伯 ㄅㄛ˙ bo5 1i7 big5
```

Comment:
I don't think we need this entry if we just want to take in 伯伯 as ㄅㄛˊ ㄅㄛ˙, since you can still type ㄅㄛ˙ alone and so it's not an "invalid" syllable.